2025/04/09 13:32:38 Logging started...
2025/04/09 13:32:38 Logging started...
2025/04/09 13:32:38 Application is starting...
2025/04/09 13:32:38 Database connection established.
2025/04/09 13:32:38 Seeding database with predetermined words...
2025/04/09 13:32:38 Database seeding completed.
2025/04/09 13:32:38 Datadog tracer started
2025/04/09 13:32:38 Go application started successfully
2025/04/09 13:32:38 Starting server on :8080...
2025/04/09 13:41:20 Datadog Tracer v1.69.1 ERROR: failure sending traces (attempt 1), will retry: Post "http://localhost:8136/v0.4/traces": dial tcp [::1]:8136: connect: connection refused (occurred: 09 Apr 25 13:40 EDT)
2025/04/09 13:41:20 Datadog Tracer v1.69.1 ERROR: lost 1 traces: Post "http://localhost:8136/v0.4/traces": dial tcp [::1]:8136: connect: connection refused (occurred: 09 Apr 25 13:40 EDT)
